nav/nav_model/nav_model_stars.py

```
@@ -307,11 +304,11 @@ def _stars_list_for_obs(self,
 
         star_list3: list[MutableStar] = []
-        for star, (ra_pm, dec_pm),u, v, u1, v1, u2, v2 in zip(star_list2,
-                                              ra_dec_pm_list,
-                                              u_list, v_list,
-                                              u1_list, v1_list,
-                                              u2_list, v2_list):
+        for star, (ra_pm, dec_pm), u, v, u1, v1, u2, v2 in zip(star_list2,
+                                                               ra_dec_pm_list,
+                                                               u_list, v_list,
+                                                               u1_list, v1_list,
+                                                               u2_list, v2_list):
```

Ruff (0.14.7)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)